security/s2a/internal/crypter/common.go

```diff
@@ -0,0 +1,15 @@
+package crypter
+
+// S2AAeadCrypter is the interface for an AEAD cipher used by the S2A record protocol.
```

Comment:
Even though there's no line limit in Go, can we try to keep the comments to 80 characters per line? This makes them easy to read :)

Reply:
Done. Do you suggest the same for code as well?

Reply:
No. We don't need to impose any line length for the code, just for the comment since they're texts. The idea is if there's a paragraph of comment, and if the lines are long, they would break in a way that would make the comment hard to read.